### PR TITLE
Incorrect namespace prefix in example table

### PR DESCRIPTION
**Pull request by Netflyerr**

### CODE REVIEW

**Review by drupol — APPROVED** [review]

**Review by shadowhand — CHANGES_REQUESTED** [review]
This change is not correct.

File: accepted/PSR-4-autoloader.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ class name, namespace prefix, and base directory.
 | Fully Qualified Class Name    | Namespace Prefix   | Base Directory           | Resulting File Path
 | ----------------------------- |--------------------|--------------------------|-------------------------------------------
 | \Acme\Log\Writer\File_Writer  | Acme\Log\Writer    | ./acme-log-writer/lib/   | ./acme-log-writer/lib/File_Writer.php
-| \Aura\Web\Response\Status     | Aura\Web           | /path/to/aura-web/src/   | /path/to/aura-web/src/Response/Status.php
+| \Aura\Web\Response\Status     | Aura\Web\Response  | /path/to/aura-web/src/   | /path/to/aura-web/src/Response/Status.php
```
Comment (shadowhand) [inline accepted/PSR-4-autoloader.md:65]: If the prefix was set to `Aura\Web\Response` then the resulting file path for `Aura\Web\Response\Status` would be `src/Status.php`.

**Review by drupol — COMMENTED** [review]
Indeed, sorry, the change is not correct.

File: accepted/PSR-4-autoloader.md (hunk 1 above)
Comment (drealecs) [reply accepted/PSR-4-autoloader.md:65]: Exactly. Existing prefix is inline with the Aura.Web configured composer psr-4 mapping: https://github.com/auraphp/Aura.Web/blob/2.x/composer.json#L22

**Pull request closed without merging**